internal/prefixcollector/collector_file_output_test.go

```diff
@@ -112,14 +148,14 @@ func (eps *ExcludedPrefixesSuite) testCollectorWithFileOutput(ctx context.Contex
 		eps.T().Fatalf("Error transforming yaml to prefixes: %v", err)
 	}
 
+	eps.Require().LessOrEqual(modified, len(sources)*2)
```

Comment:
How does we calculate the modification count?

Also, seems like actual and expected should be swaped (see at input params of the function)

Reply:
I've chosen this value because for each file write there may be two `nsnotify.Write` events maximum. So for each source we need to have at least two possible write events.